src/components/Logo.tsx

```diff
@@ -0,0 +1,43 @@
+import { ComponentPropsWithRef, HTMLAttributes, forwardRef } from 'react'
+
+import PluralLogoFull from './icons/logo/PluralLogoFull'
+import PluralLogoMark from './icons/logo/PluralLogoMark'
+import PluralLogoWord from './icons/logo/PluralLogoWord'
+
+import { scaling } from './LoopingLogo'
+
+export type LogoProps = ComponentPropsWithRef<'div'> & {
+  isDark?: boolean
+  scale?: number;
+  type?: LogoType;
+}
+
+export enum LogoType {
+  Full = 'full',
+  Word = 'word',
+  Mark = 'mark',
+}
+
+const LoopingLogo = forwardRef<HTMLDivElement, LogoProps & HTMLAttributes<HTMLDivElement>>(({
+  isDark = false,
+  scale,
```

Comment:
I'd rather use width and height if possible. With transform it might be harder to style it.